core/com.b2international.snowowl.core.rest/snow-owl-api-docs/src/App.js

```diff
@@ -0,0 +1,64 @@
+import 'antd/dist/antd.css';
+import 'swagger-ui-react/swagger-ui.css';
+
+import React from 'react';
+import SwaggerUI from "swagger-ui-react"
+import { Layout, BackTop, Menu } from 'antd';
+
+const { Content, Sider } = Layout;
+
+class App extends React.Component {
+
+  state = {
+    selectedKey: 'admin'
+  }
+
+  onMenuSelect = (e) => {
+    this.setState({
+      selectedKey: e.key
+    })
+  }
+
+  render() {
+    return (
+      <>
+        <BackTop />
+        <Layout>
+          <Sider
+            style={{
+              overflow: 'auto',
+              height: '100vh',
+              position: 'fixed',
+              left: 0,
+            }}
+          >
+            <div className="logo">
+              <img src={`${process.env.PUBLIC_URL}/logo-snow-owl.png`} alt="logo-snow-owl" height="60" style={{maxWidth:"100%"}} />
+            </div>
+            <Menu
+              onSelect = { this.onMenuSelect }
+              defaultOpenKeys={['sub1']}
```

Comment:
It looks like this property is only needed if at least one nested `SubMenu` is present.